### PR TITLE
[stable/moodle] Chart for moodle

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Comment by sameersbn** [comment]
fixed: 
 - command to fetch password from secrets volume
 - updated locked mariadb chart version
 - remove mariadb persistent volume `storageClass` default

File: stable/moodle/.helmignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.git
```
Comment (lachie83) [inline stable/moodle/.helmignore]: This isn't the standard .helmignore. Did you generate using `helm create`

**Comment by lachie83** [comment]
@sameersbn Thanks for the contrib. This LGTM

File: stable/moodle/.helmignore (hunk 1 above)
Comment (seanknox) [reply stable/moodle/.helmignore]: Here's the standard `.helmignore` you can use:

```
# Patterns to ignore when building packages.
# This supports shell glob matching, relative path matching, and
# negation (prefixed with !). Only one pattern per line.
.DS_Store
# Common VCS dirs
.git/
.gitignore
.bzr/
.bzrignore
.hg/
.hgignore
.svn/
# Common backup files
*.swp
*.bak
*.tmp
*~
# Various IDEs
.project
.idea/
*.tmproj
```

**Review by seanknox — CHANGES_REQUESTED** [review]

**Review by seanknox — APPROVED** [review]

**Pull request merged**